coreos-4/policies/RA-Risk_Assessment/component.yaml

```diff
@@ -117,34 +117,33 @@
 - control_key: RA-5
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: partial
+  implementation_status: not applicable
   narrative:
     - key: a
       text: |
-        This control reflects organizational procedures/policies, and is not
-        applicable to the configuration of CoreOS.
+        Organizational procedures to initially scan for vulnerabilities and to perform a scan
+        when new vulnerabilities are identified and reported are outside the scope of RHCOS
+        configuration.
     - key: b
       text: |
-        A control response is planned. Progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-352
+        Organizational procedures to facilitate interoperability and vulnerability management
+        process automation are outside the scope of RHCOS configuration.
     - key: c
       text: |
-        This control reflects organizational procedures/policies, and is not
-        applicable to the configuration of CoreOS.
+        Organizational procedures to analyze vulnerability scan outputs from security controls
+        are outside the scope of RHCOS configuration.
     - key: d
       text: |
-        A control response is planned. Progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-352
+        Organizational procedures to remediate legitimate vulnerabilties within organization
+        determined response times is outside the scope of RHCOS configuration.
```

Comment:
For RA-5(d) a response detailing how to remediate vulnerabilities natively in OpenShift would be appropriate. For example Clair/Quay at the registry level or OpenSCAP during CI/CD.

Reply:
I added language supporting Clair/Atomic remediation.
The Clair and Atomic scanners can be hosted in OpenShift, but could be replaced by other products within an organization's vulnerability management program.